src/tests/BuildWasmApps/testassets/LibraryWithResources/LibraryWithResources.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Should we be targeting 5.0 here?

Reply:
Tests+net6.0 will be fixed in the workloads PR.